### DISCUSSION TITLE
Architecture?

### DISCUSSION THREAD

**Discussion by MNLierman** [inv2004/ttop#36] [General]
It would help to know for those who come across your project what architecture your static build is built for, if it's built for one architecture or several. Unless I am missing something obvious, this is not mentioned anywhere. I searched on Google and Bing to make sure that I wasn't crazy in assuming that builds must be targeted to a specific architecture (and hadn't progressed past that requirement).

**Comment by inv2004** [comment]
Thank you for pointing into it. It is only `x86_64` at the moment.

Just mentioned it here: https://github.com/inv2004/ttop/blob/main/README.md#static-binary-x86-64
